lib/builders/dataTypes/listViewBuilder/listViewPropertyBuilder.ts

```diff
@@ -0,0 +1,39 @@
+﻿import {ListViewDataTypeBuilder} from "../listViewDataTypeBuilder";
+
+export class ListViewPropertyBuilder {
+  parentBuilder: ListViewDataTypeBuilder;
+  alias: string;
+  header: string;
+  nameTemplate: string;
```

Comment:
When the property "nameTemplate" is used? as I don't see it in json payload.

Reply:
It's in the columns displayed, if you add a value under "Enter a label template"